docs/self-managed/identity/user-guide/connect-to-an-existing-keycloak.md

```diff
@@ -0,0 +1,68 @@
+---
+id: connect-to-an-existing-keycloak
+title: "Connect to an existing Keycloak instance"
+sidebar_label: "Connect to an existing Keycloak instance"
+---
+
+In this guide, we'll demonstrate how to connect the Identity component to your existing Keycloak instance.
+
+### Prerequisites
+
+- Access to your [Keycloak Admin Console](https://www.keycloak.org/docs/16.1/server_admin/#using-the-admin-console)
+- A basic understanding of [administering realms and clients](https://www.keycloak.org/docs/16.1/server_admin/#assembly-managing-clients_server_administration_guide) in Keycloak
+
+### Steps
+
+To connect Identity to an existing Keycloak instance, take the following steps:
+
+1. Log in to your Keycloak Admin Console.
+
+2. Select the realm you would like to connect Identity to. In our example, this is **Test Realm**
+
+![keycloak-admin-realm-select](img/keycloak-admin-realm-select.png)
+
+3. Select **Clients** in the navigation menu, and click the **Create** button to create a new client.
+
+![keycloak-admin-client-list](img/keycloak-admin-client-list.png)
+
+4. Enter the client ID and the URL of where your Identity instance will be hosted and click **Save**.
+   :::note What client ID should I use?
+   By default, Identity uses the Client ID `camunda-identity`, so we recommend using this too. If you choose a different client ID, this will need to be set in the Identity application [environment variables](/docs/self-managed/identity/deployment/configuration-variables.md).
+   :::
+
+![keycloak-admin-client-add](img/keycloak-admin-client-add.png)
+
+5. On the page for the client created, set the **Access Type** to `confidential`, **Service Accounts Enabled** to `ON`, and save your changes by clicking the **Save** button.
+
+![keycloak-admin-update-client-1](img/keycloak-admin-update-client-1.png)
+
+6. Navigate to the **Service Account Roles** tab in the top navigation.
+
+![keycloak-admin-update-client-2](img/keycloak-admin-update-client-2.png)
+
+7. Select the `realm-management` client from the **Client Roles** dropdown.
+
+![keycloak-admin-update-client-3](img/keycloak-admin-update-client-3.png)
+
+8. Assign the `manage-clients`, `manage-realm`, and `manage-users` role from the **Available Roles** list.
+
+![keycloak-admin-update-client-4](img/keycloak-admin-update-client-4.png)
+
+:::note Why does Identity need these roles?
+Identity is designed to allow users to manage the various entities related to the Camunda Platform. To achieve this, it requires specific access to the realm.
+::: 9. Navigate to the **Credentials** tab and copy the client secret.
```

Comment:
```suggestion
:::note Why does Identity need these roles?
Identity is designed to allow users to manage the various entities related to the Camunda Platform. To achieve this, it requires specific access to the realm.
::: 

9. Navigate to the **Credentials** tab and copy the client secret.
```

😆 I was so confused looking at the docs locally, wondering where step 9 went! Found it!

Reply:
Ah yea, I spotted this locally, unfortunately with images and note blocks my IDE doesn't play well with the numbering in the list :/

Reply:
Ah no worries. When this get's merged it's a good thing to double check.